roles/openshift_named_certificates/tasks/main.yml

```diff
@@ -73,6 +73,7 @@
   ignore_errors: True
   register: ca_bundle_result
   when: named_certificates_ca_file_defined | length > 0
+  failed_when: false
```

Comment:
Since you are using `failed_when` you can drop `ignore_errors`.  You can also add `changed_when: false` so it does not look like it is changing something, since it is not.

Reply:
Yeah, I will drop the `ignore_errors`.